Packs/GLIMPS_Detect/Integrations/GLIMPSDetect/GLIMPSDetect.py

```diff
@@ -283,21 +282,9 @@ def main():
 
     # Log exceptions
     except GDetectError as e:
-        resp = client.gclient.response
-        if isinstance(resp.error, BadAuthenticationToken):
-            error = f'Given token has bad format: {resp.message}'
-        elif isinstance(resp.error, NoAuthenticateToken):
-            error = f'No token to authentication exists: {resp.message}'
-        elif isinstance(resp.error, NoURL):
-            error = f'No URL to API found: {resp.message}'
-        elif isinstance(resp.error, UnauthorizedAccess):
-            error = f'Access to API is unauthorized: {resp.message}'
-        elif isinstance(resp.error, BadUUID):
-            error = f'Given UUID is wrong: {resp.message}'
-        else:
-            error = f'GDetectError: {str(e)}'
+        return_error(str(e))
```

Comment:
I would do:
 `return_error(GDetectError: {str(e)})`
as before the changes.

Reply:
`GDetectError` has an[ informative prefix description](https://github.com/glimps-re/py-gdetect/blob/e61a87589f580703fad5c330599c0127b46aaed0/src/gdetect/exceptions.py#L16), perhaps 'GDetectError:'  prefix is not that usefull